src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const populationElements = document.querySelectorAll('.population');
+
+const populations = Array.from(populationElements).map((element) => {
+  return parseInt(element.textContent.replace(/,/g, ''), 10);
+});
+
+const totalPopulation = populations.reduce(
+  (sum, population) => sum + population,
+  0,
+);
+const avaragePopulation = Math.round(totalPopulation / populations.length);
```

Comment:
There is a typo in the variable name 'avaragePopulation'. It should be 'averagePopulation'.